### PR TITLE
Database dynamic name

### PR DESCRIPTION
**Pull request by ArturMinnullin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - Support [Heroku Review Apps](https://devcenter.heroku.com/articles/github-integration#review-apps)
 - Update [rails](https://github.com/rails/rails) version up to 4.2.3
 - Add [Rails ERD](https://github.com/voormedia/rails-erd) gem for generating a diagram based on application's AR models
+- Implement the dynamic database name
```
Comment (timurvafin) [inline CHANGELOG.md]: Please put this line on top of section.

**Comment by timurvafin** [comment]
Looks :ok:, except one note. Please fix it and squash commits, after that we will be ready to merge.
Thank you for the awesome job!

**Comment by ArturMinnullin** [comment]
@timurvafin done

**Comment by artembiserov** [comment]
I think if we have DATABASE_NAME variable, then development db name and test db name will be the same

**Pull request merged**